ci/docker/runtime_functions.sh

```diff
@@ -1125,6 +1125,30 @@ nightly_tutorial_test_ubuntu_python2_gpu() {
     nosetests-3.4 --with-xunit --xunit-file nosetests_tutorials.xml test_tutorials.py --nologcapture
 }
 
+nightly_example_test_ubuntu_python3_gpu() {
+    set -ex
+    cd /work/mxnet/docs
+    export BUILD_VER=tutorial
+    export MXNET_DOCS_BUILD_MXNET=0
+    make html
+    export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
+    export MXNET_TEST_KERNEL=python3
+    cd /work/mxnet/tests/examples
+    nosetests-3.4 --with-xunit --xunit-file nosetests_tutorials.xml test_examples.py --nologcapture
```

Comment:
Does this PR depend on https://github.com/apache/incubator-mxnet/pull/13271 to be merged?

Reply:
as mentioned here - https://github.com/apache/incubator-mxnet/pull/13271#discussion_r233745860, please combine the PRs